README.md

```diff
@@ -1 +1,14 @@
 # MediaMonks - Frontend Coding Standards
+
+## CSS
+
+We use [Sass](https://sass-lang.com/) as css preprocessor.
+
+#### Selectors
+Only use class names for styling, no id's.
```

Comment:
I would phrase this as:
Preferably use classes for selectors. Ideally we avoid using html elements as selectors, unless they are used to set base styles. In some cases it can be allowed to target data-attributes. Never use ids as a selector without a good reason.